src/bin/stratisd.rs

```diff
@@ -19,6 +21,8 @@ use std::error::Error;
 use std::rc::Rc;
 use std::cell::RefCell;
 use std::process::exit;
+use std::os::unix::io::AsRawFd;
```

Comment:
Please alphabetize your imports. In theory, we alphabetize them everywhere within sub categories of standard lib, external crate, our crates , imports within stratisd. This one clearly escaped, but no need to exacerbate the problem.